sonar-db/src/main/java/org/sonar/db/permission/template/PermissionTemplateDao.java

```diff
@@ -69,6 +69,10 @@ public int countUserLoginsByQueryAndTemplate(DbSession session, PermissionQuery
     return mapper(dbSession).selectUserPermissionsByTemplateIdAndUserLogins(templateId, Collections.emptyList());
   }
 
+  /**
+   * @deprecated does not support organizations. Should return group ids.
+   */
+  @Deprecated
   public List<String> selectGroupNamesByQueryAndTemplate(DbSession session, PermissionQuery query, long templateId) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)